### DISCUSSION TITLE
charp(CS8604) Possível parametron de referencia nula para o parametro.

### DISCUSSION THREAD

**Discussion by isaac0m** [balta-io/forum#305] [Tira Dúvidas] [answered]
Boa tarde a Todos!

Sou novo por  aqui  e a pouco tempo comecei a aprender o C#, e no curso Fundamentos, na aula de Mão na Massa Editor de Texto me ocorre um problema quando vou executar o programa. 

Aparece a mensagem (charp(CS8604) Possível parametron de referencia nula para o parametro.), isso ocorre na seguinte expressão:  short opcao = short.Parse(Console.ReadLine());

O mesmo código no Visual Studio não gera nenhum erro, mas no VS Code isso sempre acontece.  Tentei me virar sozinho mas ainda não consegui resolver.
Seguem duas imagens no VS Code e no visual studio 2022.

![VsCode](https://user-images.githubusercontent.com/46767592/231537619-bddd55d7-b98e-4e3f-af25-bd1c76bbf431.png)
![Viisual Studio](https://user-images.githubusercontent.com/46767592/231537654-6b92ed8e-3dee-41ed-8057-a07224f9dd7e.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @isaac0m como vai?
Seja bem vindo!

Este vídeo explica:
https://www.youtube.com/watch?v=PNasRBbo00g&t=947s

**Comment by isaac0m** [comment]
Boa tarde!
Balta, isso esclareceu todas as minhas duvidas, obrigado pelos seus ensinamentos. Muito ansioso para finalizar todos os seus cursos.
